authorization-service/authorization_service/lambda_func/basic_authorizer.py

```diff
@@ -0,0 +1,62 @@
+import os
+import base64
+
+
+def lambda_handler(event, context):
+    print(f"event: {event}")
+
+    authorization_header = event.get('authorizationToken', '')
```

Comment:
In case identity source of authorizer is configured correctly auth lambda will not be called if `Authorization` header is missing, so there's no need in this code.

Reply:
I'm not sure how it works for Token authorizer, but in case of Request authorizer if `Authorization`'s header value contains spaces only it will be trimmed and auth lambda won't be called (401 http status is returned)